src/_vmprof.c

```diff
@@ -148,10 +148,11 @@ static PyObject *enable_vmprof(PyObject* self, PyObject *args)
 {
     int fd;
     int memory = 0;
+    int lines = 0;
     double interval;
     char *p_error;
 
-    if (!PyArg_ParseTuple(args, "id|i", &fd, &interval, &memory))
+    if (!PyArg_ParseTuple(args, "id|ii", &fd, &interval, &memory, &lines))
```

Comment:
The parameters are named in the vmprof.enable method, so there is no urge to make them named here. If it is possible I would avoid making this more complex.